### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (462)
--This is the signup page for new users so that they can create an ID and then enjoy the site.

### CODE REVIEW

File: site-pages/sign-up/sign.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.signup-main{
+    display: flex;
+    margin-top: 30px;
+    margin-bottom: 30px;
```
Comment (dev-enforced) [inline site-pages/sign-up/sign.css]: Can use margin:30px 0; instead of this

File: site-pages/sign-up/sign.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    align-items: center;
+    justify-content: center;
+}
+.signup-form-container{
+    box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
+    align-self: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```
Comment (dev-enforced) [inline site-pages/sign-up/sign.css]: Can create a common class in one of the files that is root.css and pass it to the appropriate elements in your HTML file.

File: site-pages/sign-up/sign.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    text-align: center;
+    padding: 20px;
```
Comment (dev-enforced) [inline site-pages/sign-up/sign.css]: Can use rems here too

**Review by dev-enforced — APPROVED** [review]
The code looks good to me !! Gave some minor suggestions that you can add to your code to save your time while making changes later

File: site-pages/sign-up/sign.css (hunk 1 above)
Comment (Jia-A) [reply site-pages/sign-up/sign.css]: Oh okay, shorthand property. Will change it.

File: site-pages/sign-up/sign.css (hunk 2 above)
Comment (Jia-A) [reply site-pages/sign-up/sign.css]: Yes.

File: site-pages/sign-up/sign.css (hunk 3 above)
Comment (Jia-A) [reply site-pages/sign-up/sign.css]: I totally forgot about changing px to rem. Will do that.

**Review by MonikaJhingan — APPROVED** [review]
The layout is looking good.

**Pull request merged**